specification/components/schemas/CodeSystemConcept.yaml

```diff
@@ -35,7 +35,7 @@ properties:
       | Clinic Type                     | General Urology               |
       | Appointment Cancellation Reason | Inappropriate service         |
       | Referral Cancellation Reason    | Intend to go private          |
-      | Appointment Non-attendance      | Patient ill                   |
+      | Appointment Non-attendance Reason     | Patient ill                   |
       
```

Comment:
I know it renders correctly, but just for neatness, could you align these

Reply:
I've still got branch checked out so will align